openadr-client/src/lib.rs

```diff
@@ -411,25 +412,6 @@ impl Client {
             .collect())
     }
 
-    /// Get a single program from the VTN that matches the given target
-    pub async fn get_program(&self, target: Target<'_>) -> Result<ProgramClient> {
-        let pagination = PaginationOptions { skip: 0, limit: 2 };
-
-        let mut programs = self
-            .get_programs_request(
-                Some(target.target_label()),
-                target.target_values(),
-                pagination,
-            )
-            .await?;
-
-        match programs[..] {
-            [] => Err(crate::Error::ObjectNotFound),
-            [_] => Ok(programs.remove(0)),
-            [..] => Err(crate::Error::DuplicateObject),
-        }
-    }
-
```

Comment:
the only target that you can give that gives a single result is ProgramName, so we can just use a `get_program_by_name`